server/service/appconfig.go

```diff
@@ -23,9 +23,9 @@ import (
 	"github.com/kolide/kit/version"
 )
 
-// //////////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////
 // Get AppConfig
-// //////////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
That was just to touch a Go file to trigger the CI Go test suite.

Reply:
Looks good. We can also add `.github/workflows/test-go.yaml` to the `paths:` in `.github/workflows/test-go.yaml` (Basically to trigger a run if it's itself modified.).

Reply:
Yes! It's actually already there, but I wanted to make sure I didn't miss any other task that would require changes.